lib/events/test/suite.go

```diff
@@ -166,11 +169,22 @@ func (s *EventsSuite) EventPagination(c *check.C) {
 // SessionEventsCRUD covers session events
 func (s *EventsSuite) SessionEventsCRUD(c *check.C) {
 	// Bob has logged in
+<<<<<<< HEAD
```

Comment:
Bad merge?

Reply:
@Tener Yup, also needed to rebase. Resolved now.